app/api/push-subscriptions/route.ts

```diff
@@ -5,14 +5,14 @@ import {
 import { NextRequest, NextResponse } from 'next/server';
 import webPush, { PushSubscription } from 'web-push';
 
-webPush.setVapidDetails(
-  'mailto:test@example.com',
-  process.env.NEXT_PUBLIC_NOTIFICATION_PUBLIC_KEY ?? '',
-  process.env.NOTIFICATION_PRIVATE_KEY ?? ''
-);
-
 export async function POST(request: NextRequest) {
   try {
+    webPush.setVapidDetails(
```

Comment:
This is a temporary fix. Having this piece of code outside the route handler caused the build to fail due to the environment variables not being available during build-time.